test/tools/libtesteth/ImportTest.cpp

```diff
@@ -289,11 +296,18 @@ json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o, boo
 			str = value.get_str();
 		else if (value.type() == json_spirit::array_type)
```

Comment:
I don't understand.  `postParseJson()` does recursion.  So `makeAllFieldsHex` should not.  Otherwise two different functions traverse the same branch of JSON.  That's confusing.